src/Footer.js

```diff
@@ -1,22 +1,158 @@
+                <a href="#">Press Releases</a>
+                </li>
+                <li className="leading-5">
+                <a href="#">Amazon Cares</a>
+                </li>
+                <li className="leading-5">
+                <a href="#">Gift a Smile</a>
+                </li>
+                <li className="leading-5">
+                <a href="#">Amazon Science</a>
+                </li>
+            </ul>
+            </div>
+        <div className="sec">
+          <h5 className="foot-head">
+            Contact Us
+          </h5>
+          <ul className="text-gray-200 text-base">
+            <li className="leading-5">
+              <a href="#">Facebook</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Twitter</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Instagram</a>
+            </li>
+          </ul>
+        </div>
+        <div className="sec">
+          <h5 className="foot-head">Make Money with Us</h5>
+          <ul className="text-gray-200 text-base">
+            <li className="leading-5">
+              <a href="#">Sell products on Amazon</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Sell apps on Amazon</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Become an Affiliate</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Advertise Your Products</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Self-publish with Us</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Host an Amazon Hub</a>
+            </li>
+          </ul>
+        </div>
+        <div className="sec">
+          <h5 className="foot-head">Let Us Help You</h5>
+          <ul className="text-gray-200 text-base">
+            <li className="leading-5">
+              <a href="#">Amazon and COVID-19</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Your Account</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Your Orders</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Shipping Rates and Policies</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Returns and Replacements</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Manage Your Content and Devices</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Amazon Assistant</a>
+            </li>
+            <li className="leading-5">
+              <a href="#">Help</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div className='footer-line'>
+      </div>
+        <div className="rest-footer-2">
+            <img src="" alt="logo"></img>
```

Comment:
Hey @ramyashreeshetty, I can see here that in the src section you have not assigned any image, Is that so?

Reply:
I had assigned it. It wasn't working so cleared it out.

Reply:
I tried again it isnt working, i'll drop the image over here.

![amazon-logo](https://user-images.githubusercontent.com/43513353/191677219-e5ab38ab-df01-490b-bc6c-10fc0556e81c.png)